### DISCUSSION TITLE
New version? With new icon?

### DISCUSSION THREAD

**Discussion by michael1006** [transmission/transmission#2095] [Q&A]
Just curious if there is any estimate of when a new version might be on the way, since 3.0.0 was a year and a half ago?

It's tiny, but I'm really just wanting an updated icon, which was committed six months ago:

https://github.com/transmission/transmission/pull/1650

Seeing as the icon update was for the Big Sur style and now we've moved to Monterey, I'm just hoping we might see something soon. :)

(I'm aware I can manually replace the icon in Finder, but it would be nice to have it be the default.)

**Comment by PeterNjeim** [comment]
You can always build the project yourself using the instructions in the readme to get the latest version. They won't release a new version unless things are stabilized.
